azure-out-of-cluster.md

```diff
@@ -0,0 +1,98 @@
+Kubecost provides the ability to view Kubernetes metrics side-by-side with external cloud services cost, e.g. Azure Database Services. Additionally, it allows Kubecost to reconcile spend with your actual Azure bill. This gives teams running Kubernetes a complete and accurate picture of costs. [More info on this functionality](http://blog.kubecost.com/blog/complete-picture-when-monitoring-kubernetes-costs/)
+
+To configure out-of-cluster (OOC) costs for Azure in Kubecost, you just need to set up daily exportation of cost reports to Azure storage. Once cost reports are exported to Azure Storage, Kubecost will access them through the Azure Storage API to display your OOC cost data alongside your in cluster costs.
+
+## Step 1: Export Azure Cost Report
+
+Follow this guide taking note of the name that you use for the exported cost report and select the daily month-to-date option for how the reports will be created.
+
+https://docs.microsoft.com/en-us/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal
+
+It will take a few hours to generate the first report, after which Kubecost can use the Azure Storage API to pull that data. 
+
+>note: If you have sensitive data in an Azure Storage account and do not want to give out access to it, create a separate Azure Storage account to store your cost data export.
+
+## Step 2: Provide Access to Azure Storage API
+
+The values needed to provide access to the Azure Storage Account where cost data is being exported can be found in the Azure portal in the Storage account where the cost data is being exported. 
+* `<STORAGE_ACCOUNT_NAME>` is the name of the Storage account where the exported CSV is being stored.
+* `<STORE_ACCESS_KEY>` can be found by selecting the “Access Keys” option from the navigation sidebar  then selecting “Show Keys”. Using either of the two keys will work. 
+* `<REPORT_CONTAINER_NAME>` is the name that you choose for the exported cost report when you set it up. This is the name of the container where the CSV cost reports are saved in your Storage account. 
+
+<details>
```

Comment:
The dropdown is cool but even knowing that you can do this in GH markdown I missed it the first time I read through and was confused. I would suggest keeping everything out of a dropdown.